bridge/client/app/app.routing.ts

```diff
@@ -38,23 +37,6 @@ const routes: Routes = [
     component: ProjectBoardComponent,
     children: [
       { path: '', pathMatch: 'full', component: KtbEnvironmentViewComponent },
-      {
-        path: 'uniform',
-        component: KtbUniformViewComponent,
-        children: [
-          { path: 'services', component: KtbKeptnServicesListComponent },
-          { path: 'services/:integrationId', component: KtbKeptnServicesListComponent },
-          { path: 'services/:integrationId/subscriptions/add', component: KtbModifyUniformSubscriptionComponent },
-          {
-            path: 'services/:integrationId/subscriptions/:subscriptionId/edit',
-            component: KtbModifyUniformSubscriptionComponent,
-          },
-          { path: 'secrets', component: KtbSecretsListComponent },
-          { path: 'secrets/add', component: KtbCreateSecretFormComponent },
-          { path: '', pathMatch: 'full', redirectTo: 'services' },
-        ],
-      },
-      { path: 'integration', component: KtbIntegrationViewComponent },
```

Comment:
Please keep this URIs for backwards-compatibility and do a redirect.

e.g. `{ path:'/project/:projectName/uniform/services', redirectTo:'/project/:projectName/settings/uniform/integrations'},`

Reply:
After discussion with @Kirdock we agreed to not add backwards-compability, since this links are usually not saved and also not logged by other tools.